source/tests/system/nirfmxnr_session_tests.cpp

```diff
@@ -118,6 +118,7 @@ TEST_F(NiRFmxNRSessionTest, TwoInitializedSessionsOnSameDevice_CloseSessions_Clo
   EXPECT_EQ(0, init_response_one.status());
   EXPECT_TRUE(status_two.ok());
   EXPECT_EQ(0, init_response_two.status());
+  EXPECT_NE(init_response_one.instrument().id(), init_response_two.instrument().id());
```

Comment:
This seemed like a good pre-req for the test and mirrored the new test I added below. Besides that, though, this test should cover Case 2 in the [Session Management part of the HLD](https://ni.visualstudio.com/DevCentral/_git/ASW?path=/Specs/Hardware%20Configuration/E1665864%20gRPC%20API%20for%20RFmx/F1433350%20gRPC%20API%20for%20RFmx%20SpecAn.md&_a=preview&anchor=session-management).